portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -63,9 +65,11 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
       long timestamp = (long) entity.getProperty("timestamp");
       String message = (String) entity.getProperty("message");
       FormSubmissions comment = new FormSubmissions(id, fname, lname, email, number, timestamp, message);
```

Comment:
In a design perspective, when should we post comments on the website. What if someone doesn't actually enter anything, but press the submit button? What if they don't write a message?